tmt/convert.py

```diff
@@ -635,9 +678,13 @@ def read_nitrate(beaker_task, common_data, disabled, general):
     return common_data, individual_data
 
 
-def read_nitrate_case(testcase, makefile_data=None, general=False):
+def read_nitrate_case(testcase: TestCase,
+                      makefile_data: Optional[Dict[str,
```

Comment:
I am thinkin about creating aliases also for `Dict[str, Any]` the code would become more readable

Reply:
Added, thanks